hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -xeo pipefail
 
-OCP_VERSION=${OCP_VERSION:-4.16.0-rc.2}
+OCP_VERSION=${OCP_VERSION:-4.17.0-rc.2}
```

Comment:
This is used in upstream CI end-to-end lanes. I'd avoid changing this, or we'll have conflicts during sync merge.

Reply:
removed this change. PTAL